src/smartpeak/source/ui/Workflow.cpp

```diff
@@ -78,7 +78,7 @@ namespace SmartPeak
                     "QUANTIFY_FEATURES",
                     "STORE_FEATURES" };
           else if (s_string == "HPLC UV Unknowns")
-            ids = { "LOAD_RAW_DATA"
+            ids = { "LOAD_RAW_DATA",
```

Comment:
Interesting bug, compiler in this case decides to concatenate two strings instead of failing...